composer.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "symfony/polyfill",
-    "type": "library",
+    "type": "metapackage",
```

Comment:
actually, because of polyfill-util, the root package should stay as a library, and its autoload section should list Util.